typings/react-native-keychain.d.ts

```diff
@@ -72,6 +72,10 @@ declare module 'react-native-keychain' {
     securityLevel?: SECURITY_LEVEL;
     storage?: STORAGE_TYPE;
     rules?: SECURITY_RULES;
+    promptInfoTitle?: string;
```

Comment:
I'd really find it preferable for the options to live in their own object and not mix in with the other options. ie.

```
promptOptions: {
  title?: string,
  subtitle?: string,
  // ... etc
}
```

Reply:
I though about doing that, but I didn't now what style you wanted, so I just mixed them with the existing props. I'll fix this tomorrow.

Reply:
@vonovak Done